.pre-commit-config.yaml

```diff
@@ -1,4 +1,9 @@
 repos:
+  - repo: https://github.com/hadialqattan/pycln
+    rev: v2.0.2 # must match requirements-tests.txt
+    hooks:
+      - id: pycln
+        args: [--all, .]
```

Comment:
Maybe limit this to `stubs`, `stdlib`, and `scripts`, to be safe.

Reply:
`tests`, as well?

Reply:
Sure.